libmpv/opengl_cb.h

```diff
@@ -163,10 +179,22 @@ extern "C" {
  *
  * glMPGetNativeDisplay("wl") should return a Wayland "struct wl_display *".
  *
- * glMPGetNativeDisplay("drm") should return a DRM FD casted to intptr_t (note
- * that a 0 FD is not supported - if this can happen in your case, you must
- * dup2() it to a non-0 FD).
- *
+ * glMPGetNativeDisplay("drm-params") should return an mp_drm_params structure pointer:
+ */
+struct mp_drm_params {
+    // DRM fd (int). Note that a 0 FD is not supported - if this
+    // can happen in your case, you must dup2() it to a non-0 FD).
+    int fd;
```

Comment:
There's no need for that 0 special case. It was sort of needed with the old openglcb/drm API, because NULL was the default return value for "not present". Define -1 as invalid/not-present/unused value instead.